posts/2022-03-11-dev-mode-with-eclipse.adoc

```diff
@@ -0,0 +1,101 @@
+
+https://openliberty.io/docs/latest/development-mode.html[Liberty dev mode] enables developers to quickly and iteratively develop cloud-native Java applications with the latest Jakarta EE and MicroProfile technologies. Dev mode will watch your project for file changes and provide hot reload and deployment, on demand testing, and debugger support. You can work with dev mode from anywhere, whether it's from a terminal with a text editor or your favorite IDE. https://marketplace.eclipse.org/content/ibm-liberty-developer-tools[Liberty Developer Tools] is an Eclipse plugin that provides helpful server configuration editor views. In the following sections we detail the current best practice for using dev mode, and optionally Liberty Developer Tools, to develop your application within the Eclipse IDE.
+
+Check out a demo of working with Liberty dev mode in the Eclipse IDE:
+
+++++
+<iframe width="560" height="315" src="https://www.youtube.com/embed/613VBYdk6f8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+++++
+
+== Working with dev mode in the Eclipse IDE with Liberty Developer Tools installed
+
+If you have Liberty Developer Tools installed in your Eclipse workspace, the server configuration editor views can be helpful as you modify your `server.xml` and other configuration files.
+
+image::/img/blog/liberty-dev-tools-server-config-view.png[Liberty Developer Tools server configuration view for the Eclipse IDE,width=70%,align="center"]
+
+Dev mode handles a lot of the heavy lifting for developers when using the Liberty runtime. With dev mode, you no longer need to manage your Liberty instances through the "Servers" view of Liberty Developer Tools.
+Disable Liberty Developer Tools' integration for Maven and Gradle projects:
+
+* For Maven projects: **Window -> Preferences -> Maven -> Liberty Integration** drop-down to "No".
+
+* For Gradle projects: **Window -> Preferences -> Gradle -> Liberty Integration** drop-down to "No".
+
+image::/img/blog/liberty-dev-tools-disable-servers-view.png[Liberty Developer Tools disabling the Servers view for the Eclipse IDE,width=70%,align="center"]
+
+== Setting up the project
+
+For Maven projects, specify the Liberty Maven Plugin in your project's `pom.xml` file:
+[source,xml]
+----
+    <plugin>
+        <groupId>io.openliberty.tools</groupId>
+        <artifactId>liberty-maven-plugin</artifactId>
+        <version>3.5.1</version>
+    </plugin>
+----
+
+For Gradle projects, specify the Liberty Gradle Plugin in your project's `build.gradle` file:
+[source,groovy]
+----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.3'
+    }
+}
+----
+
+== Interacting with dev mode
```

Comment:
shouldn't have two headings back to back- I thibnk you could remove the "Interacting..." heading since all the content is under subheadings and just add "with Dev Mode" to the subheadings